### ISSUE TITLE
unconstrained type parameter is null compiles when not targeting C# 8

### ISSUE THREAD

**Issue by YairHalberstadt** [dotnet/roslyn#35550]
**Version Used**: VS 16.0.3

**Steps to Reproduce**:

Compile the following program with LangVersion set to 7.3

```csharp
public class C {
    public bool M<T>(T t) {
        return t is null;
    }
}
```

**Expected Behavior**: Error: Cannot convert null to type parameter 'T' because it could be a non-nullable value type.

**Actual Behavior**: Compiles without error.

**Comment by jcouv** [comment]
Yes, the `t is null` should be gated on a language version.  This was fixed in 16.1 (PR https://github.com/dotnet/roslyn/pull/34695)
I'll go ahead and close as fixed.
FYI @gafter

**Comment by YairHalberstadt** [comment]
I see it's a duplicate now - sorry about that. GitHub search is far from perfect.
